packages/docbook_xsl_nons.rb

```diff
@@ -0,0 +1,101 @@
+require 'package'
+
+class Docbook_xsl_nons < Package
+  description 'The DocBook XSL Stylesheets package contains XSL stylesheets. These are useful for performing transformations on XML DocBook files.'
+  compatibility 'all'
```

Comment:
Can we put this line below version?  I know order doesn't matter but it helps with reviews to have consistency.

Reply:
done